README.md

```diff
@@ -62,6 +62,71 @@ def chat_demo():
 
 `https://<your_app_name>.skygeario.com/static/demo/index.html`
 
+### Sending push notification
+
+Push notification can be implemented by the following code.
+Please ensure APNS certificate and private key are properly setup,
+if you are using [Skygear.io], you can configure it at the setting panel.
+
+```python
+from skygear.container import SkygearContainer
+from skygear.options import options as skyoptions
+from skygear.action import push_user
+
+# Please modify these import statment according to your project structure
```

Comment:
You can assume that skygear-cms is installed as a package (using pip/setup.py) so the package name will be `chat.*`.

You can also make the comment better by saying in what scenario should developer use relative imports (“If you installed the plugin using git submodule, add `.` to use relative import.”)